EIPS/eip-1.md

```diff
@@ -3,7 +3,8 @@ eip: 1
 title: EIP Purpose and Guidelines
 status: Active
 type: Meta
-author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>
+author: Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, and others
```

Comment:
Please change this to:
```
author: "Martin Becze <mb@ethereum.org>, Hudson Jameson <hudson@ethereum.org>, and others:  
        https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1.md"
```
Testing that locally, it doesn't render the newline since they are stripped, but it does show the colon. We could fix up the rendering of the link and newline later after this is merged.